handler/http_client.go

```diff
@@ -83,13 +80,13 @@ func (h *Handler) checkResponseBody(resourceName string, expectedBody *gherkin.D
 		return err
 	}
 
-	if err := compare.Value(actual, expected); err != nil {
-		b := bytes.NewBufferString("\nJSON mismatch (" + err.Error() + ")\n\n")
-		t := tablewriter.NewWriter(b)
-		compare.Print(t, "", actual, expected)
-		t.Render()
-		return errors.New(b.String())
+	comparison, err := compare.JSON(body, []byte(expectedBody.Content))
```

Comment:
this would break the current test that behave as this function is `contains` instead of `exact compare` right?

Reply:
Added contains and equals comparisons